server/model/avg.py

```diff
@@ -0,0 +1,37 @@
+#import labels from model.GeminiClient
+from model.GeminiClient import POLITICAL_LABELS
+
+#political labels manually listed out
+#POLITICAL_LABELS = ["communist", "leftist", "liberal", "moderate", "conservative", "nationalist"]
+
+def avg_of_objects(dicts_of_json: dict) -> dict:
+#labels to dynamically create dictionary 
+    labels = POLITICAL_LABELS
+    sub_labels = ["total_magnitude", "avg_magnitude", "percent_of_whole"]
+    default_value = 0
+#Ceate dictionary to return
+    weighted_avg = {
+        label: {sub_label: default_value for sub_label in sub_labels} for label in labels
+    }
+#Total Magnitude of input tracked to determine percentage of whole for each label
+    total_sum_of_magnitudes = 0
+#This is necessary if the inputted dictionary will have a key as in the example output of the gemini client
+    dict_to_search = dicts_of_json["example_output"]

```

Comment:
i don't think "example_output" will be in the actual output.

the input would be a list of comments with labels/magnitudes